File: python/ex095.py
# Refaça o exercício 90, só que agora em forma de função Somador(), que vai
# receber dois parâmetros e vai retornar o resultado da soma entre eles para o
# programa principal.

def SomadorPlus(numberOne, numberTwo):
    return print(f"Resultado: {numberOne + numberTwo}")

def Main():
    numberOne = int(input("Digite um numero: "))
    numberTwo = int(input("Digite outro numero: "))
    
    return SomadorPlus(numberOne, numberTwo)


Main()

File: typescript/src/ex095.ts
// Refaça o exercício 90, só que agora em forma de função Somador(), que vai
// receber dois parâmetros e vai retornar o resultado da soma entre eles para o
// programa principal.

var readline = require('readline-sync');


function SomadorMaster(valueOne: number, valueTwo: number) {
    const soma = valueOne + valueTwo;
    return soma;
}

function Mainster(){
    let valueOne:number = parseInt(readline.question("Digite um valor: "));
    let valueTwo:number = parseInt(readline.question("Digite outro valor: "));

    console.log(SomadorMaster(valueOne,valueTwo));
}

Mainster();
